go/vt/mysqlctl/backupengine.go

```diff
@@ -112,7 +112,7 @@ type BackupRestoreEngine interface {
 var BackupRestoreEngineMap = make(map[string]BackupRestoreEngine)
 
 func init() {
-	for _, cmd := range []string{"mysqlctl", "mysqlctld", "vtcombo", "vttablet", "vttestserver", "vtctld", "vtctldclient", "vtexplain", "vtbackup"} {
+	for _, cmd := range []string{"mysqlctld", "vtcombo", "vttablet", "vttestserver", "vtctld", "vtctldclient", "vtbackup"} {
```

Comment:
vtctldclient can be removed from this list.

Reply:
mysqlctld should also be removed.

Reply:
done